apps/web/src/application/services/ListNetworks/ListNetworksService.ts

```diff
@@ -0,0 +1,28 @@
+import { NetworkRepository } from '../../../domain/repositories/NetworkRepository'
+import { ObservableService } from '../../ObservableService'
+import { ListNetworksResponseDto } from './ListNetworksResponseDto'
+
+
+export class ListNetworksService extends ObservableService<void, ListNetworksResponseDto> {
+  private readonly networkRepository: NetworkRepository
+
+  constructor(
+    networkRepository: NetworkRepository,
+  ) {
+    super()
+    this.networkRepository = networkRepository
+  }
+
+  protected async process(): Promise<ListNetworksResponseDto> {
+    const networks = await this.networkRepository.list()
+    const selectedNetwork = await this.networkRepository.getSelected()
```

Comment:
It seems these 2 requests are independent, so they could run in parallel, saving some time

```suggestion
    const [networks, selectedNetwork] = await Promise.all([
      this.networkRepository.list()
      this.networkRepository.getSelected()
   ])
```

Reply:
Done. Thanks!